src/site/mobile.css

```diff
@@ -31,6 +31,15 @@ pre {
   padding: 10px;
 }
 
+.page-link {
```

Comment:
Link is not grayed out on first and last page:

![image](https://user-images.githubusercontent.com/4871350/127058012-f23091e5-3f4d-47b2-9a64-eef6f45dab58.png)

Reply:
Removed buttons altogether for first and last pages in 07e34dc